backend/src/main/java/co/inventorsoft/scripty/service/ProjectServiceInterface.java

```diff
@@ -0,0 +1,10 @@
+package co.inventorsoft.scripty.service;
+
+import co.inventorsoft.scripty.model.dto.ProjectToFront;
+import org.springframework.security.core.Authentication;
+
+import java.util.Collection;
+
+public interface ProjectServiceInterface {
```

Comment:
I clearly see that this is an interface from the declaration. You don't need to put this information inside name